packages/api-docs/src/PageJsonGenerator.tsx

```diff
@@ -157,11 +158,7 @@ function createPageJsonFiles(collectedData: CollectedData, options: IPageJsonOpt
 }
 
 function renderDefaultValue(section: DocNodeContainer): string {
-  let defaultValueAsString = '';
-  for (const node of section.nodes) {
-    defaultValueAsString += extractText(node);
-  }
-  return defaultValueAsString;
+  return section.nodes.map((node: DocNode) => extractText(node)).join('');
```

Comment:
nit, but you can probably get away with

```suggestion
  return section.nodes.map(extractText).join('');
```

Reply:
Agree